orc8r/gateway/python/magma/common/cert_validity.py

```diff
@@ -100,12 +100,13 @@ def cert_is_invalid(host, port, certfile, keyfile, loop):
 
     # Invalid cert only when tcp succeeds and ssl fails
     if isinstance(ssl_res, Exception):
-        logging.error(
-            'Error making SSL connection: %s, %s',
+        logging.debug(
+            'Tcp connection succeeded but SSL connection did not: %s, %s. Assuming invalid SSL certificate.',
```

Comment:
```suggestion
            'TCP connection succeeded but SSL connection did not: %s, %s. Assuming invalid SSL certificate.',
```
For consistency. :wink: